demo/mobile/src/app/modules/user/index.js

```diff
@@ -0,0 +1,5 @@
+angular.module('wfm.user', []);
```

Comment:
might be easier to just put sync and this inside `wfm.common.apiservices` to see if it works.